integration_test/src/test_runner/test_generator_util.rs

```diff
@@ -0,0 +1,168 @@
+        expected_column_name.push(column_name);
+
+        let display_size = get_column_attribute(&stmt, i, Desc::DisplaySize, &NUMBER_VAL)?;
+        expected_display_size.push(display_size);
+
+        let length = get_column_attribute(&stmt, i, Desc::Length, &NUMBER_VAL)?;
+        expected_length.push(length);
+
+        let is_searchable = get_column_attribute(&stmt, i, Desc::Searchable, &NUMBER_VAL)?;
+        expected_is_searchable.push(is_searchable);
+
+        let is_unsigned = get_column_attribute(&stmt, i, Desc::Unsigned, &NUMBER_VAL)?;
+        expected_is_unsigned.push(is_unsigned);
+
+        let sql_type = get_column_attribute(&stmt, i, Desc::Type, &NUMBER_VAL)?;
+        expected_sql_type.push(sql_type);
+
+        let bson_type = get_column_attribute(&stmt, i, Desc::TypeName, &STRING_VAL)?;
+        expected_bson_type.push(bson_type);
+
+        let precision = get_column_attribute(&stmt, i, Desc::Precision, &NUMBER_VAL)?;
+        expected_precision.push(precision);
+
+        let scale = get_column_attribute(&stmt, i, Desc::Scale, &NUMBER_VAL)?;
+        expected_scale.push(scale);
+
+        let nullability = get_column_attribute(&stmt, i, Desc::Nullable, &NUMBER_VAL)?;
+        expected_nullability.push(nullability);
+    }
+
+    // 2. Get result set data
+    let mut expected_result: Vec<Vec<Value>> = vec![];
+
+    while fetch_row(&stmt)? {
+        let mut row: Vec<Value> = vec![];
+        for i in 0..(column_count) {
+            let expected_data_type = get_expected_data_type(expected_sql_type.get(i).unwrap());
+            let field = get_data(&stmt, i as USmallInt, expected_data_type)?;
+            row.push(field);
+        }
+        expected_result.push(row);
+    }
+
+    // 3. Create new TestEntry with all the data
+    let test_entry = TestEntry {
+        description: entry.description.clone(),
+        db: entry.db.clone(),
+        test_definition: entry.test_definition.clone(),
+        expected_result: Some(expected_result),
+        skip_reason: None,
+        ordered: entry.ordered,
+        expected_catalog_name: Some(expected_catalog_name),
+        expected_case_sensitive: Some(expected_case_sensitive),
+        expected_column_name: Some(expected_column_name),
+        expected_display_size: Some(expected_display_size),
+        expected_length: Some(expected_length),
+        expected_is_searchable: Some(expected_is_searchable),
+        expected_is_unsigned: Some(expected_is_unsigned),
+        expected_sql_type: Some(expected_sql_type),
+        expected_bson_type: Some(expected_bson_type),
+        expected_precision: Some(expected_precision),
+        expected_scale: Some(expected_scale),
+        expected_nullability: Some(expected_nullability),
+    };
+
+    // 4. Write the TestEntry to a file
+    let now = std::time::SystemTime::now()
+        .duration_since(std::time::UNIX_EPOCH)
+        .unwrap()
+        .as_millis();
```

Comment:
The JDBC driver's test generator uses a nice `MMddHHmmss` date format, but date formatting in rust is a bit of a headache and requires importing external crates (such as `chrono` which has security issues or `time` which is not intuitive) so I decided to just go with milliseconds instead. It is doubtful people will retain generated files over time so the timestamp portion of these file names is not very important anyway, but lmk if you have opinions on changing this.

Reply:
This is good.  
The important part is that it won't overwrite the already generated files and it sorts well to see the latest files which epoch time does.